bin/SyncDBData.rb

```diff
@@ -320,25 +404,9 @@ def sync
       begin
         counter += 1
         if file_exists(copy_from, $src_fs) then
```

Comment:
```suggestion
        if file_exists("#{copy_from}.gz", $src_fs) then
```
I need to do it in two steps because the interface doesn't allow suggestions for those code deletions.
But, we will primarily use `gzips` and advice this in general, so checking for gzip first make more sense to me.